ext/opentelemetry-ext-pymongo/src/opentelemetry/ext/pymongo/__init__.py

```diff
@@ -77,6 +78,7 @@ def started(self, event: monitoring.CommandStartedEvent):
                 self._remove_span(event)
 
     def succeeded(self, event: monitoring.CommandSucceededEvent):
+        """ TODO """
```

Comment:
"Method to handle a pymongo CommandSucceededEvent"

Reply:
```suggestion
        """ Method to handle a pymongo CommandSucceededEvent """
```